app/scripts/services/dimSettingsService.factory.js

```diff
@@ -83,6 +84,15 @@
       // self destruct timer for quality, hope we can remove this some day...
       savedSettings.disableQuality = new Date('9/19/2016') <= new Date();
 
+      // for now just override itemTags. eventually let users create own?
```

Comment:
I think we absolutely need custom tags.  Give people's imagination to mark up their content.  They will surprise you.  

Also, filtering by tags is more accurate than filtering by notes because of spelling errors.  With tags, you get a drop down with fixed items and a icon to verify.  

Tags are the way to go.